src/main/java/com/exadel/sandbox/officeFloorAttachment/controllers/service/impl/AttachmentServiceImpl.java

```diff
@@ -0,0 +1,75 @@
+package com.exadel.sandbox.officeFloorAttachment.controllers.service.impl;
+
+import com.exadel.sandbox.officeFloorAttachment.controllers.dto.AttachmentDto;
+import com.exadel.sandbox.officeFloorAttachment.controllers.entities.Attachment;
+import com.exadel.sandbox.officeFloorAttachment.controllers.exception.EntityNotFoundException;
+import com.exadel.sandbox.officeFloorAttachment.controllers.repositories.AttachmentRepository;
+import com.exadel.sandbox.officeFloorAttachment.controllers.service.AttachmentService;
+import lombok.RequiredArgsConstructor;
+import org.modelmapper.ModelMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Component
+@RequiredArgsConstructor
+public class AttachmentServiceImpl implements AttachmentService {
+
+    private final ModelMapper mapper;
+
+    @Autowired
+    AttachmentRepository attachmentRepository;
+
+    @Override
+    public List<AttachmentDto> getAttachments() {
+        List<Attachment> all = attachmentRepository.findAll();
+
+        List<AttachmentDto> list = new ArrayList<>();
+
+        for (Attachment attachment: all){
+            list.add(mapper.map(attachment, AttachmentDto.class));
+        }
+        return list;
```

Comment:
Please rewrite this method something like that: 
     List<Attachment> all = attachmentRepository.findAll();

        return all.stream()
                .map(attachment -> mapper.map(attachment, AttachmentDto.class))
                .collect(Collectors.toList());

And also please check other places and try to use stream where possible